src/agents/supervisor/prompts.py

```diff
@@ -1,112 +1,52 @@
 - Focus solely on the key points raised in the query.
 
-Sample Queries and Responses:
-- General queries that are irrelevant to Kyma or Kubernetes:
-  Query: "What is the capital of France?"
-  Output:
-  ```
-  {{
-    "response": "Paris"
-  }}
-  ```
-
-- Kyma or Kubernetes related queries:
-  Query: "What is Kyma function?"
-  Output:
-  ```
-  {{
-      "subtasks": [
-          {{
-              "description": "What is Kyma function?",
-              "assigned_to": "KymaAgent"
-          }}
-      ]
-  }}
-  ```
-
-  Query: "What is Kyma serverless? Why my k8s deployment is failing?"
-  Output:
-  ```
-  {{
-      "subtasks": [
-          {{
-              "description": "What is Kyma serverless?",
-              "assigned_to": "KymaAgent"
-          }},
-          {{
-              "description": "Why my k8s deployment is failing?",
-              "assigned_to": "KubernetesAgent"
-          }}
-      ]
-  }}
-  ```
+Agent Classification:
+- "KymaAgent": Manages Kyma specific topics
+- "KubernetesAgent": Handles Kubernetes related queries
+- "Common": Covers all other general queries
 
-  Query: "Why my k8s deployment is failing?"
-  Output: 
-  ```
-  {{
-    "subtasks": [
-        {{
-            "description": "Why my k8s deployment is failing?",
-            "assigned_to": "KubernetesAgent"
-        }}
-    ]
-  }}
-  ```
-
-  Query: "How to create a Python app that performs Discounted Cash Flow (DCF) calculations? How to create a Kyma function? How to create a k8s service for this function?"
-  Output: 
-  ```
-  {{
-    "subtasks": [
-        {{
-            "description": "How to create a Python app that performs Discounted Cash Flow (DCF) calculations?",
-            "assigned_to": "Common"
-        }},
-        {{
-            "description": "How to create a Kyma function?",
-            "assigned_to": "KymaAgent"
-        }},
-        {{
-            "description": "How to create a k8s service for this function?",
-            "assigned_to": "KubernetesAgent"
-        }}
-    ]
-  }}
-  ```
 
 Kyma terminologies: Kyma, Kubernetes, Serverless, Service Mesh, API Gateway, API Rule, Istio, Service Catalog, Application Connector, Eventing, Telemetry, Tracing, Logging, Kyma Runtime, module, Service Management.
 
 Kubernetes terminologies: Pod, Node, Cluster, Namespace, Container, Deployment, ReplicaSet, Service, Ingress, ConfigMap, Secret, Volume, PersistentVolume, PersistentVolumeClaim, StatefulSet, DaemonSet, Job, CronJob, HorizontalPodAutoscaler, NetworkPolicy, ResourceQuota, LimitRange, ServiceAccount, Role, RoleBinding, ClusterRole, ClusterRoleBinding, CustomResourceDefinition, Operator, Helm Chart, Taint, Toleration, Affinity, InitContainer, Sidecar, Kubelet, Kube-proxy, etcd, Kube-apiserver, Kube-scheduler, Kube-controller-manager, Container Runtime.
 
-Remember: The goal is to create a plan that directly reflects the original query without elaboration or additional steps.
```

Comment:
let's test.

Reply:
I tested with all possible prompt with multple run , this  prompt work as expected , it redundant to add same info again in the prompt